dash/frontend/src/generic-styles.scss

```diff
@@ -116,17 +131,44 @@ $classesToGenerate: (
       inherit: inherit, initial: initial, revert: revert,
       revert-layer: "revert-layer", unset: unset
     )
+  ),
+  overflow: (
+    abbreviation: 'overflow',
+    values: (
+      auto: 'auto',
+      visible: 'visible',
+      scroll: 'scroll',
+      hidden: 'hidden'
+    )
+  ),
+  overflow-x: (
+    abbreviation: 'overflow-x',
+    values: (
+      auto: 'auto',
+      visible: 'visible',
+      scroll: 'scroll',
+      hidden: 'hidden'
+    )
+  ),
+  background-color: (
```

Comment:
Is this being used? I didn't see it in use, but maybe I missed it.

Reply:
@becky-intelletive  we have used this in cluster list and image scan result pages